code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -27,6 +27,18 @@
 	required_reagents = list("orangejuice" = 1, "cornoil" = 1)
 	result_amount = 2
 
+/datum/chemical_reaction/skub
+	name = "skub"
+	id = "skub"
+	result = null
+	required_reagents = list("spraytan" = 1, "liquidgibs" = 1)
```

Comment:
I don't like the idea of explaining what skub is, just make the clown spawn with it